### ISSUE TITLE
JSON POST data to API

### ISSUE THREAD

**Issue by Lunar-Arc** [evepraisal/go-evepraisal#113]
Hey!

I am writing an application in C# (dotnet core) that consumes your public API endpoints but I am struggling to understand what format is required for the POST data.

I can currently send the raw string data to your API to be parsed in the regex and materialise the results into my application with the following example: 
![image](https://user-images.githubusercontent.com/27502419/68396523-9a4aab00-0169-11ea-9c83-5f633eb3c8be.png)

However, I need to be able to parse the results in my application before sending them on to your endpoint because I would like to filter out undesirable assets first. I do this by performing a regex on the string, materialising the results into object collections and then filtering out the groups/categories.
![image](https://user-images.githubusercontent.com/27502419/68397124-9703ef00-016a-11ea-8f83-a910fbe415d4.png)

Rather than attempting to reconstruct a string that would match your regex based on the materialised assets in my AssetItem type, I would like to send the collection to be parsed by your API as JSON data in a POST request, but I seem to be struggling with formatting the request in such a way that your API does not return a 400 bad request.

In short, what format can I send POST data that will allow your API to parse the results of multiple items.
If I send `{ avatar }` I get a single result of 1x Avatar.
If I send `{ avatar, avatar}` I get a single result of 2x Avatar.
If I send `{ "name": "avatar", "quantity": 2 }` I get a 400 bad request.
![image](https://user-images.githubusercontent.com/27502419/68398499-bbf96180-016c-11ea-9ae3-e033bb3e1e1f.png)

**Comment by Lunar-Arc** [comment]
Never mind, I managed to find a suitable solution.
